parsons/google/google_sheets.py

```diff
@@ -370,6 +375,10 @@ def overwrite_sheet(
                 Otherwise, values will be entered as strings or numbers only.
         """
 
+        if not len(table.columns) and not table.num_rows:
+            logger.warning("No data provided to overwrite sheet, skipping.")
+            return
+
```

Comment:
I don't think this is appropriate, if no data is returned that may be a valid result and should be allowed to overwrite a sheet.

Reply:
I guess that is weird if the table has no columns, no idea what circumstances that would come up.

Reply:
Thanks for reviewing! Your comment made me look a bit closer at this and I realized that the error I mentioned is happening _after_ the columns are written anyway (which, you're right, the table should basically alway have) when the `cells` list passed to `update_cells` is empty.

I just changed this so:

- The sheet is always cleared.
- If there are no columns a warning is logged and it stops. Obviously this would almost never happen but no harm in handling it.
- If there are no rows a warning is logged and it finishes normally.